Netskope improvements (#37493)
demisto/content: Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector_description.md

   - Network
   - Alert
   - Page

> - Note: The collector can handle 35K events per minute on average per each event type.  

Reviewer: ```suggestion
- Note: The collector can handle up to 35K events per minute on average. 
```
